src/responsiveStore.js

```diff
@@ -3,10 +3,25 @@ import MediaQuery from 'mediaquery'
 import {transform, keys} from 'lodash'
 
 /**
- * @arg {object} breakpoints - Hash of custom breakpoints.
+ * @arg {object} [options]  - Hash of custom breakpoints.
  * @returns {class} Responsive store class to be passed to `alt.createStore`.
  */
-export default function create_responsive_store_class(breakpoints) {
+// export the factory (but not as default)
+export function create_responsive_store(options) {
+    // use `options` as hash of breakpoints
+    let breakpoints = options
+    // if `options` was not provided
+    if (typeof options === 'undefined') {
```

Comment:
Can't this just be replaced with `if(!breakpoints)` since the object is truth-y true?

We could have a line underneath that says something like breakpoints = breakpoints ? breakpoints : default_breakpoints